actions/class.RestResource.php

```diff
@@ -132,6 +132,7 @@ public function getAll()
                 $search   = $this->hasRequestParameter('search') ? $this->getRawParameter('search') : '';
                 $limit    = $this->hasRequestParameter('limit') ? $this->getRequestParameter('limit') : 30;
                 $offset   = $this->hasRequestParameter('offset') ? $this->getRequestParameter('offset') : 0;
+                $excludedClasses = $this->getExcludedClasses();
```

Comment:
```suggestion
                $excludedClasses = $this->hasGetParameter('exclude') ? $this->getExcludedClasses() : '';
```
And then remove if with `$this->hasGetParameter('exclude')` from `getExcludedClasses()`

Reply:
I'm not sure if this is more clean version.